### PR TITLE
Feature/dicom web support

### PR DESCRIPTION
**Pull request by hummerichsander**

### CODE REVIEW

File: adit/batch_transfer/tests/integration/test_batch_transfer.py [hunk 1]
```diff
@@ -40,3 +40,38 @@ def test_unpseudonymized_urgent_batch_transfer_succeeds(
...
+    page.get_by_label("Source").select_option(label="DICOM Server Orthanc Test Server 1")
+    page.get_by_label("Destination").select_option(label="DICOM Server Orthanc Test Server 2")
+    page.get_by_label("Start transfer urgently").click(force=True)
+    page.get_by_label("Project name").fill("Test transfer")
+    page.get_by_label("Project description").fill("Just a test transfer.")
+    page.get_by_label("Ethics committee approval").fill("I have it, I swear.")
+    page.get_by_label("Batch file").set_input_files(files=[batch_file])
+    page.locator('input:has-text("Create job")').click()
+    expect(poll(page.locator('dl:has-text("Success")'))).to_be_visible()
+    # page.screenshot(path="foo.png")
```
Comment (medihack) [inline adit/batch_transfer/tests/integration/test_batch_transfer.py:77]: Let's delete this debugging oddment

File: adit/token_authentication/tests/integration/test_token_authentication.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    response = requests.get(
+        channels_live_server.url + "/token-authentication/test",
+        headers={"Authorization": f"Token {token_str}"},
+    )
+    assert response.status_code == 200
+
+    page.on("dialog", lambda dialog: dialog.accept())
+    page.query_selector('[id*="delete-token-button"]').click()  # type: ignore
+    page.reload()
+    time.sleep(5)
```
Comment (medihack) [inline adit/token_authentication/tests/integration/test_token_authentication.py:35]: Do we really need to sleep 5 seconds here? Sounds pretty long.

Comment (medihack) [reply adit/token_authentication/tests/integration/test_token_authentication.py:35]: Why do we have to sleep at all? Are there any race conditions? Is the token delete by an AJAX request?

Comment (hummerichsander) [reply adit/token_authentication/tests/integration/test_token_authentication.py:35]: indeed, there is a Ajax Request for the token delete view. Nevertheless, a shorted sleep time probably would be sufficient

File: adit/batch_transfer/tests/integration/test_batch_transfer.py (hunk 1 above)
Comment (hummerichsander) [reply adit/batch_transfer/tests/integration/test_batch_transfer.py:77]: What exactly?

Comment (medihack) [reply adit/batch_transfer/tests/integration/test_batch_transfer.py:77]: Just the commented out line ;-)

**Pull request merged**